packages/itwin/grouping-mapping-widget/src/widget/components/Grouping.tsx

```diff
@@ -425,25 +433,27 @@ export const Groupings = ({ mapping, goBack }: GroupsTreeProps) => {
           <Surface className='gmw-groups-container'>
             <div className='gmw-toolbar'>
               <DropdownMenu
-                className='gmw-ui-provider-dropdown'
+                className='gmw-custom-ui-dropdown'
                 disabled={isLoadingQuery}
                 menuItems={() =>
-                  (customUIs.length > 0
-                    ? customUIs
+                  (groupUIs.length > 0
+                    ? groupUIs
                     : defaultUIMetadata)
                     .map((p) => (
                       <MenuItem
                         key={p.name}
                         onClick={() => addGroup(p.name)}
                         icon={p.icon}
                         className='gmw-menu-item'
+                        data-testid={`gmw-add-${p.name}`}
```

Comment:
Are there any restrictions of what can be in the `data-testid` attribute? Can it contain spaces? Can the `name` contain spaces? Are all characters allowed in both of them?

Reply:
the name has a validator not allowing spaces in the UI